examples/storage/s3/incremental_learning/README.md

```diff
@@ -15,17 +20,13 @@ metadata:
 stringData: # use `stringData` for raw credential string or `data` for base64 encoded string
   ACCESS_KEY_ID: XXXX
   SECRET_ACCESS_KEY: XXXXXXXX
+EOF
 ```
 
-### Attach the created secret to the Model/Dataset/Job.
-`EDGE_NODE` and `CLOUD_NODE` are custom nodes, you can fill it which you actually run.  
-
-```
-EDGE_NODE="edge-node" 
-CLOUD_NODE="cloud-node"
-```
-
-* Attach the created secret to the Model.  
+### Prepare Model
+* Download [models](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/model.tar.gz).  
+* Put the unzipped model file into the bucket of your cloud storage service.  
+* Attach the created secret to the Model and create Model.  
 
 ```yaml

```

Comment:
ditto, update `yaml` to `shell` or `sh` or `bash`

Reply:
fix it